app/config/FeatureSwitches.scala

```diff
@@ -19,17 +19,16 @@ package config
 import play.api.Configuration
 import routing.Version
 
-case class FeatureSwitch(maybeConfig: Option[Configuration]) {
+case class FeatureSwitches(featureSwitchConfig: Configuration) {
```

Comment:
This also needs the `object FeatureSwitches....` from individuals-income-received-api

Reply:
Also FeatureSwitchesSpec needs to be copied over

Reply:
Fixed